freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -468,6 +468,20 @@ public DnsZone addReverseDnsZone(String cidr) throws FreeIpaClientException {
         return (DnsZone) invoke("dnszone_add", flags, params, DnsZone.class).getResult();
     }
 
+    public DnsRecord getDnsRecord(String dnsZoneName, String recordName) throws FreeIpaClientException {
+        List<String> flags = List.of(dnsZoneName, recordName);
+        Map<String, Object> params = Map.of();
+        return (DnsRecord) invoke("dnsrecord_show", flags, params, DnsRecord.class).getResult();
+    }
+
+    public DnsRecord addDnsCnameRecord(String dnsZoneName, String recordName, String cnameRecord) throws FreeIpaClientException {
+        LOGGER.error("ASDJ CNAME " + dnsZoneName + " -- " + recordName + "  -- " + cnameRecord + " -- ");
```

Comment:
why error? `asdj`?

Reply:
I removed this log statement. The caller of this method logs debug messages.